wikimedia/wikipedia-ios
# [ALT TEXT] Add tooltips

**Phabricator:** https://phabricator.wikimedia.org/T370364

### Notes
This PR adds tooltips to Alt Text.

### Test Steps
1. Fresh install, enable developer setting flags.
2. Go through image recommendations flow, publish edit.
3. Tap Add on alt text prompt.
4. Tap Continue on alt text onboarding
5. Confirm tooltips appear as expected.
